tests/model_registry/rbac/conftest.py

```diff
@@ -89,6 +92,77 @@ def sa_token(service_account: ServiceAccount) -> str:
         raise
 
 
+@pytest.fixture(scope="function")
+def new_group(
+    request: pytest.FixtureRequest,
+    admin_client: DynamicClient,
+    test_idp_user_session: UserTestSession,
+) -> Generator[str, None, None]:
+    """
+    Fixture to create a new OpenShift group and add a user, then delete the group after the test.
+    The group name is passed as a parameter to the fixture and the user is taken from test_idp_user_session.
+    """
+
+    group_name = request.param
+    with Group(
+        client=admin_client,
+        name=group_name,
+        users=[test_idp_user_session.username],
+        wait_for_resource=True,
+    ) as _:
+        LOGGER.info(f"Group {group_name} created successfully.")
+        yield group_name
+        LOGGER.info(f"Group {group_name} deletion initiated by context manager.")
+
+
+@pytest.fixture(scope="function")
+def model_registry_group_with_user(
+    admin_client: DynamicClient,
+    test_idp_user_session: UserTestSession,
+) -> Generator[Group, None, None]:
+    """
+    Fixture to manage a test user in the Model Registry group.
+    Adds the user to the group before the test, then removes them after.
+
+    Args:
+        admin_client: The admin client for accessing the cluster
+        test_idp_user_session: The test user session containing user information
+
+    Yields:
+        Group: The Model Registry group with the test user added
+    """
+    model_registry_users_group = f"{MR_INSTANCE_NAME}-users"
```

Comment:
might be worth having this parametrized rather than using the constant in case we want to expand this test for multi-tenancy and need to deal with the default groups of multiple MR instances

Reply:
+1. Tests should pass the group name.